test/parallel/test-http-res-write-end-dont-take-array.js

```diff
@@ -23,41 +23,37 @@
 require('../common');
 const assert = require('assert');
 const http = require('http');
+const common = require('../common');
```

Comment:
See line 23 -- common has already been required, but just not set to a variable.

Reply:
@maclover7 : Thanks for the feedback. I've updated the PR with the changes. Kindly review now and update the CI Checks. Thanks !